src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -41,11 +41,30 @@
 
     private static final Logger LOG = Logger.getLogger(IsmDeviceApi.class);
     private final VirtualSystemElement vs;
-
     private final TransactionControl txControl;
-
     private final EntityManager em;
 
+    private List<DeviceMemberEntity> validateDeviceMembers(Long parentId, Long vsId, Long memberId, String childName) {
```

Comment:
This is a misnomer since it does not validate anything. Just returns please rename this method to getDeviceMembers when you send your next PR.